### PR TITLE
Added polyfill for Array.findIndex()

### PR DESCRIPTION
**Pull request by zerostatic**
The MultiSelect component uses method: findIndex() This requires a polyfill  for older browsers (IE11 etc...)
#### Changelog

**New**

added `import 'core-js/modules/es6.array.find-index';` to .storybook/polyfills.js

giphy

PS, great library!

### CODE REVIEW

**Comment by joshblack** [comment]
Thanks for the catch! We might want to actually try and revert the usage of `findIndex` to something that is IE11-compat, instead of bringing on another dependency. What do you think?

**Comment by zerostatic** [comment]
Sounds good. I changed findIndex() to use a a combination of map and indexOf. Let me know if that works or if you know of a better solution. Thanks!

File: src/components/MultiSelect/MultiSelect.js [hunk 1]
```diff
@@ -194,18 +194,17 @@ export default class MultiSelect extends React.Component {
...
+                      var itemIndex = selectedItem
+                        .map(function(selected) {
+                          return selected.id;
+                        })
+                        .indexOf(item.id);
                       const isChecked =
-                        selectedItem.findIndex(
-                          selected => item.id === selected.id
-                        ) !== -1;
+                        typeof selectedItem[itemIndex] !== 'undefined' || false;
```
Comment (asudoh) [inline src/components/MultiSelect/MultiSelect.js]: Thank you for your quick change @zerostatic! Given you mimicked `findIndex`, do you want to keep `!== -1` comparison?

**Comment by zerostatic** [comment]
Ah, good point! I changed to use `!== -1` Much cleaner.

**Comment by zerostatic** [comment]
Nice catches...Sorry, old habits!  These have been updated.

**Pull request merged**